templates/zookeeper-env.sh.j2

```diff
@@ -0,0 +1,5 @@
+{% if zookeeper_env is defined %}
```

Comment:
zookeeper_env will always be defined unless you don't assign anything. Instead check of length, of course loop will not work but lets prevent from entering it.